.github/workflows/release.yml

```diff
@@ -316,23 +230,20 @@ jobs:
           name: release-builds-windows
           path: ./dist/*.gz
 
-  publish-release:
-    # allow skipped but not failed
-    if: ${{
```

Comment:
deleted this krufty conditional because it's no longer applicable; it was added to allow the debug fork to run to completion at a time when the windows or darwin builds were either not working in that fork or took too long